backend/services/core/src/schema.graphql

```diff
@@ -62,6 +63,15 @@ type Mandate @key(fields: "id") {
   member: Member
 }
 
+type MandatesByPosition {
+  mandate: Mandate,
```

Comment:
Am I crazy or should this be a Position?

Reply:
fixed